src/main/java/com/google/devtools/build/lib/packages/StarlarkLibrary.java

```diff
@@ -332,8 +332,9 @@ public Depset depset(
               doc =
                   "A dict that maps configuration conditions to values. Each key is a "
                       + "<a href=\"$BE_ROOT/../skylark/lib/Label.html\">Label</a> or a label string"
-                      + " that identifies a config_setting, constraint_setting, or constraint_value"
-                      + " instance."),
+                      + " that identifies a config_setting or constraint_value instance. See the"
+                      + " <a href=\"$BE_ROOT/../skylark/macros.html#label-resolution-in-macros>"
```

Comment:
Oops, I found a small bug! No closing \" on the href. See https://docs-staging.bazel.build/5813/versions/main/skylark/lib/globals.html#select.

Reply:
Thanks for catching this, fixed.